### ISSUE TITLE
Error 500: Internal Server Error - 'NoneType' object has no attribute 'auth'

### ISSUE THREAD

**Issue by MacKStar21** [morpheus65535/bazarr#673]
**Describe the bug**
Receiving an Error 500: Internal Server Error when attempting to open the Bazarr UI.  Specific issue:  [AttributeError: 'NoneType' object has no attribute 'auth'].  Full error text and debug logs attached.  

**To Reproduce**
Does not happen at any reproducible frequency.  Occurs when opening the UI in (at least) Chrome and Safari.

**Screenshots**
![bazarr_auth](https://user-images.githubusercontent.com/57812790/68972027-d52b8f00-07b8-11ea-9639-db95fcdd80d0.png)

**Software (please complete the following information):**
 - Bazarr: dev 0.8.3.3
 - Radarr version: dev 0.2.0.1358
 - Sonarr version: master 2.0.0.5338
 - OS: OSX 10.14.6
 - Python: 2.7.16

**Additional context**
Debug Log:  
[bazarr debug.log](https://github.com/morpheus65535/bazarr/files/3852990/bazarr.debug.log)

**Comment by morpheus65535** [comment]
This exception isn't present in your debug log. Please make sure you can reproduce it before providing the debug log.

**Comment by cavaliunas-felipe** [comment]
Same for me, after some hours working, seems that some task makes trigger this error.
Just reinstalled and will send debug log when it starts to happen again.

**Comment by morpheus65535** [comment]
@cavaliunas-felipe I don't even see an exception in your debug log...

**Comment by MacKStar21** [comment]
It looks like the exception isn't picked up in the debug log.  I cleared my log, restarted the service, let it run until I came across the exception again, and nothing is in there.

**Comment by MacKStar21** [comment]
In the log i provided in this issue, the exception occurred about 2-3 mins before I exported. Nothing there.

**Comment by morpheus65535** [comment]
Are you both running MacOS X?

**Comment by cavaliunas-felipe** [comment]
No, my SO is Ubuntu.
Well, like @MacKStar21 said, it's not picked up in the debug log, seems to be just with UI...

**Comment by mdcollins05** [comment]
I'm also seeing this on my install. Running Debian 9. Good news is restarting Bazarr has fixed it for me. Not sure if it'll come back or not.

Bazarr Version: 0.8.3.3
Sonarr Version: 3.0.0.348
Radarr Version: 0.2.0.1358
Operating System: Linux-4.9.0-8-amd64-x86_64-with-debian-9.11
Python Version: 2.7.13

**Comment by morpheus65535** [comment]
I'm running the dev branch from source with Python 2.7.16 on Debian 10.1 since 2 days. Nothing to report here. I'll keep it running and try to see if I can finally reproduce it... :-/

**Comment by morpheus65535** [comment]
@mdcollins05 That an issue we had with 0.8.3.2. When you updated to 0.8.3.3, the automatic restart of Bazarr failed and left it in a zombie mode. It should not happen in the future report it if it does. Ty.

**Comment by altendorfme** [comment]
In version 0.8.3.3 I still had this problem;

I turned off Search Enabled Providers Simultaneously as commented above and am two days without problems.

Debian 9

**Comment by morpheus65535** [comment]
> In version 0.8.3.3 I still had this problem;
> 
> I turned off Search Enabled Providers Simultaneously as commented above and am two days without problems.
> 
> Debian 9

Ok keep me informed if it really fix your issue.

**Comment by morpheus65535** [comment]
What languages and providers are you using?

**Comment by cavaliunas-felipe** [comment]
Only Brazilian Portuguese, with LegendasTV and OpenSubtitles.

Em qua, 20 de nov de 2019 21:39, morpheus65535 <notifications@github.com>
escreveu:

**Comment by altendorfme** [comment]
> What languages and providers are you using?

All settings:
![2019-11-20_22-04-36](https://user-images.githubusercontent.com/1530032/69291543-54b6c500-0be2-11ea-8bcf-c199a4049c0a.jpg)

**Comment by cavaliunas-felipe** [comment]
I reinstalled Bazarr and disabled "Search Enabled Providers Simultaneously" as @altendorfme said. Will check if it solve the problem here.

**Comment by morpheus65535** [comment]
I've just pushed a fix in dev branch (0.8.3.4). If you update and restart Bazarr (manually), you shouldn't get this issue anymore.

**Comment by altendorfme** [comment]
Updated now, let's go now a few hours or days, if it happens again I warn you.

**Comment by cavaliunas-felipe** [comment]
I don't know if it was supposed to me to do, but I tried to update to dev branch via Bazarr UI (set development branch on settings and ran update task manually). When I did this, it start to take much time and when I refreshed page, the same error was there. 
Then I reinstalled 0.8.3.3 normally, disabled automatic update and here we are, UI still working 24 hours later. 

So, my conclusion is that on my system what's triggering the error is the Bazarr's update task.
